### PR TITLE
Fix #4109 : Disable LDPI for oppia app

### LINKED ISSUE #4109: Disable LDPI for oppia app
LDPI is generally used by watches and we should be targeting MDPI and above only.

Disable LDPI for oppia android app.

For the solution, see: https://github.com/oppia/oppia-android/issues/4109#issuecomment-1100236130.

### PR DESCRIPTION
**Pull request by deepanshu731**
Fixes #4109 Disabled LDPI for oppia app as it was not needed.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -6,6 +6,32 @@
   <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
   <uses-permission android:name="android.permission.INTERNET" />
   <uses-permission android:name="android.permission.WAKE_LOCK" />
+  <compatible-screens>
```
Comment (BenHenning) [inline app/src/main/AndroidManifest.xml:9]: Can you please point to the reference Android documentation for this as a comment? It'd be helpful for both the review & future reviewing.

I'm also thinking that we might need tests for this (to guard against unexpected screen sizes being added in the future), but it'll depend a bit on the documented Android behavior.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @deepanshu731. I took a first pass but I think that I need a bit more information before I can fully consider the changes--please take a look at my comment.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (deepanshu731) [reply app/src/main/AndroidManifest.xml:9]: @BenHenning  https://developer.android.com/guide/topics/manifest/compatible-screens-element , this is the reference i used.

**Comment by BenHenning** [comment]
Hmm. The warning on the dev article worries me. I also didn't see any way to configure this via AABs; it seems only controllable via the manifest. I'll need to think on this more before we proceed.

**Comment by deepanshu731** [comment]
Any updates @BenHenning ?

**Comment by BenHenning** [comment]
Apologies for the wait @deepanshu731. I'll follow up on this next week.

**Comment by BenHenning** [comment]
I do have some thoughts here--I'm following up on #4109.

**Comment by BenHenning** [comment]
The discussion is still ongoing in #4109; this isn't stale.

**Pull request closed without merging**